packages/strapi/lib/commands/generate-template.js

```diff
@@ -0,0 +1,91 @@
+'use strict';
+
+const { resolve, join, basename } = require('path');
+const fse = require('fs-extra');
+const chalk = require('chalk');
+const inquirer = require('inquirer');
+
+// All directories that a template could need
+const TEMPLATE_CONTENT = ['api', 'components', 'config/functions/bootstrap.js', 'data'];
+
+/**
+ *
+ * @param {string} templatePath Absolute path to template directory
+ * @param {string} rootBase Name of the root directory
+ */
+async function copyContent(templatePath, rootBase) {
+  for (const item of TEMPLATE_CONTENT) {
+    try {
+      await fse.copy(join(process.cwd(), item), join(templatePath, item));
```

Comment:
You should check it exists before copying as if it doesn't we shouldn't display an error and continue :)